config/locales/en/shared.yml

```
@@ -3,16 +3,19 @@ en:
   shared:
     partials:
       cookie_banner:
+        cookie_banner_content_html: |
+          <p class="govuk-body">We use some essential cookies to make this service work.</p>
+          <p class="govuk-body">We'd also like to use analytics cookies so we can understand how you use the service and make improvements.</p>
         title: Cookies on Apply for legal aid
         we_use_some: We use some essential cookies to make this service work.
         wed_also_lie: "We'd also like to use analytics cookies so we can understand how you use the service and make improvements."
```

Is this key name supposed to say 'like'?
```suggestion
        wed_also_like: "We'd also like to use analytics cookies so we can understand how you use the service and make improvements."
```